src/main/java/net/syscon/elite/web/config/SwaggerConfig.java

```diff
@@ -36,18 +37,16 @@ public SwaggerConfig(@Autowired(required = false) final BuildProperties buildPro
 
     @Bean
     public Docket nomisApi() {
-        final var docket = new Docket(DocumentationType.SWAGGER_2)
+        return new Docket(DocumentationType.SWAGGER_2)
                 .select()
                 .apis(RequestHandlerSelectors.withClassAnnotation(RestController.class))
                 .paths(PathSelectors.any())
                 .build()
-                .apiInfo(apiInfo());
-
-        docket.genericModelSubstitutes(Optional.class);
-        docket.directModelSubstitute(ZonedDateTime.class, Date.class);
-        docket.directModelSubstitute(LocalDateTime.class, Date.class);
-
-        return docket;
+                .apiInfo(apiInfo())
+                .genericModelSubstitutes(Optional.class)
+                .directModelSubstitute(ZonedDateTime.class, Date.class)
+                .directModelSubstitute(LocalDateTime.class, Date.class)
+                .directModelSubstitute(LocalDate.class, java.sql.Date.class);
```

Comment:
this line should have been the fix, but didn't do anything.  Leaving in, in case it is fixed properly some time.